src/iap/shared/components/carousel/carousel.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { Slide, SlideComponent } from '@shared/models/carousel.models';
+import { SlideDirective } from '../../directives/slide.directive';
```

Comment:
:point_right: :point_left: 
```suggestion
import { Component, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';

import { Slide, SlideComponent } from '@shared/models/carousel.models';
import { SlideDirective } from '../../directives/slide.directive';
```